package.json

```diff
@@ -3,6 +3,7 @@
   "version": "15.0.0-beta.7",
   "description": "Vue single-file component loader for Webpack",
   "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
```

Comment:
Can we change it to `types` field since looks like `types` is the recommendation now?
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
I used `types` before, but I find out `typings` in `vue` core [package.json](https://github.com/vuejs/vue/blob/dev/package.json#L9) and `vue-router`, should it be consistent here?

Reply:
Oh, I see. I guess it's ok to leave as `typings` then 🙂